src/app/about/About.module.css

```diff
@@ -0,0 +1,30 @@
+.about {
+    /* styles */
+    text-align: center;
+}
+
+.about__image_container {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 2rem;
+    margin-top: 2rem;
+
+}
+
+.about__title {
+    margin-bottom: 5rem;
+    font-size: 3rem;
+    font-family: fantasy;
```

Comment:
[Please Change]: Don't set a specific font - it's rare to customize the font for purposes of UI (and using a specific family over a given font is new to me). This is often a global setting, and in our case we will be using Poppins as configured in #10.